[2023-12-08 01:03:08] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn 31d981866fcafb83 127.0.0.1:56987 failed checkpoint posthandshake: Connected to self
[2023-12-08 01:03:08] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn 31d981866fcafb83 127.0.0.1:20131 failed checkpoint posthandshake: Connected to self
[2023-12-08 01:03:36] [warning] blockChain.go:507 dpchain/core/eles.(*BlockChain).CommitBlock Current NodeID:31d981866fcafb83f4a92dc5636564f161a9940e063c96b0d8109174459cea4a89407ba20a7725aee1ab9e573f401496c3c498acd1315ec8ebac19999cb14491 commit Block into LevelDB

[2023-12-08 01:03:37] [warning] synchronization.go:236 dpchain/blockSync.(*SyncProtocal).handleMsg Block synchronization: The number (30) of blocks requested by the remote node, and the actual number of blocks returned is (1) and currentHeight (0)

[2023-12-08 01:03:37] [warning] synchronization.go:236 dpchain/blockSync.(*SyncProtocal).handleMsg Block synchronization: The number (30) of blocks requested by the remote node, and the actual number of blocks returned is (1) and currentHeight (0)

[2023-12-08 01:03:40] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn 31d981866fcafb83 127.0.0.1:57185 failed checkpoint posthandshake: Connected to self
[2023-12-08 01:03:40] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn 31d981866fcafb83 127.0.0.1:20131 failed checkpoint posthandshake: Connected to self
